app/src/main/java/edu/cascadia/mobas/campusguidebook/data/dao/UserDao.java

```diff
@@ -0,0 +1,42 @@
+package edu.cascadia.mobas.campusguidebook.data.dao;
+
+import androidx.lifecycle.LiveData;
+import androidx.room.Delete;
+import androidx.room.Insert;
+import androidx.room.Query;
+import androidx.room.Update;
+
+import java.util.List;
+
+import edu.cascadia.mobas.campusguidebook.data.model.User;
+
+
+@androidx.room.Dao
+public interface UserDao {
+
+    // below method is use to
+    // add data to database.
+    @Insert
+    void insert(User model);
+
+    // below method is use to update
+    // the data in our database.
+    @Update
+    void update(User model);
```

Comment:
is there a reason these are named model instead of user?

Reply:
While it won't matter in reality because it is only used by CodeGen. it should be changed for consistency and clarity.